Packs/VirusTotal/Integrations/VirusTotalV3/README.md

```diff
@@ -2546,3 +2559,14 @@ Get analysis of a private file submitted to VirusTotal.
 >|Id|Threat Severity Level|Popular Threat Category|Threat Verdict|Status|
 >|---|---|---|---|---|---|---|
 >| example-analysis-id | HIGH | trojan | MALICIOUS | completed |
+
+## VT indicator fields
+
+3 indicator fields have been added to all indicator types:
+- **VT Engine Detections**. Number. Number of VT vendors which flagged the indicator as malicious.
+- **VT Engine Vendors**. Array. VT vendors which flagged the indicator as malicious.
+- **VT Engine Detection Names**. Array. VT detection names which flagged the indicator as malicious.
+
+To display the new fields in indicators, navigate to `Settings -> Objects Setup -> Indicators -> Types`. Select the desired indicator type, for example, `File`. Click on `Edit` and, under `Custom Fields`, choose the desired field and add the corresponding path. For instance, if you select the `VT Engine Detections` field for the `File` indicator type, add the path `File.VTVendors.EngineDetections`. This will enable the field to be populated in the indicator data.
```

Comment:
```suggestion
To display the new fields in indicators:

1. Navigate to `Settings > Objects Setup > Indicators > Types`. 
2. Select the desired indicator type, for example, `File`. 
3. Click `Edit` and, under `Custom Fields`, choose the desired field and add the corresponding path. For instance, if you select the `VT Engine Detections` field for the `File` indicator type, add the path `File.VTVendors.EngineDetections`. This will enable the field to be populated in the indicator data.
```